### ISSUE TITLE
Provider URL not consistent

### ISSUE THREAD

**Issue by Soonz** [oscarotero/Embed#402]
Hey,

I'm doing some specific adjustement depending on the URL of the provider but I did notice two "similar" URL can be different such as : 

https://www.reddit.com
https://www.reddit.com/

I could remove the "/" by myself but I think it's better if "Embed" did it to be more consistent.

**Comment by oscarotero** [comment]
I see the same result in both cases:

- https://oscarotero.com/embed/demo/index.php?url=https%3A%2F%2Fwww.reddit.com
- https://oscarotero.com/embed/demo/index.php?url=https%3A%2F%2Fwww.reddit.com%2F

**Comment by Soonz** [comment]
Yes, my example was bad, so I tried with another Reddit URL with a path after the '/' and it happens : 

https://oscarotero.com/embed/demo/index.php?url=https%3A%2F%2Fwww.reddit.com%2Fr%2FGlobalOffensive%2Fcomments%2Fjqnxoj%2Fthe_best_noscope_i_have_ever_seen%2F&settings=

**Comment by oscarotero** [comment]
In this case, the providerUrl value is returned by the Oembed API:

<img width="647" alt="imaxe" src="https://user-images.githubusercontent.com/377873/98539922-624eda00-228d-11eb-95ae-f53ae8a78cf7.png">

The aim of this library is return the values as is, without modify them. To normalize these values, you can use a different PSR-7 library that does it (or a helper function). Anyway, if I had to normalize this, I think it's a good practice to provide a `/` at the end of the url, instead removing it (personal opinion).

**Comment by Soonz** [comment]
I don't see the '/' at the end as an issue. If the providerUrl gave me an URL with the '/' at the end everytime I'd be be happy. The only thing I'm concerned about is consistency, if one time it returns and URL with the '/' and the other time an URL without it depending on how people works with it, it may cause problems / bugs.

And while I'm here, thank you for the quick answers and this amazing project.
